src/core/Akka.Remote/RemoteActorRef.cs

```diff
@@ -154,7 +154,7 @@ public override void Suspend()
             SendSystemMessage(new Akka.Dispatch.SysMsg.Suspend());
         }
 
-        /// <inheritdoc cref="IInternalActorRef"/>
```

Comment:
This is an internal CREF, which should work fine. If we're resolving to the wrong base class we just need to change that, not delete it.

Reply:
`/// <inheritdoc/>` could suffice since the base `IsLocal` has `/// <inheritdoc cref="IInternalActorRef"/>` tag?

Reply:
Yeah that's fine I think